content/en/docs/develop/core-modules/xatp.md

```diff
@@ -38,19 +38,19 @@ type XATP struct {
 
 - type: `string`
 
-Denom recommends using a prefix that matches the number of decimal places. For example, if the decimal point of CTXT is 6, use uCTXT.
+`Denom` is highly recommended using a prefix that matches the number of decimal places. e.g. CTXT, which decimal point is 6, is uCTXT.
 
 #### Token
 
 - type: `string`
 
-Token is the contract address of cw20.
+`Token` is the contract address of CW20.
 
 #### Pair
 
 - type: `string`
 
-AMM's pair contract address must be a pair that can query the ratio with a pool including XPLA.
+AMMs(Automated Market Makers)' pair contract address should be a trading pair including XPLA and be able to query the ratio of its pool.
```

Comment:
AMM(Automated Market Makers)'s

Reply:
Automated Market Makers is AMMs in short.

Reply:
Then shouldn't the `'` be removed?

Reply:
dont you mean it to be the possessive case? if you meant it, the apostrophe should be placed at the end of `s`.